core/src/main/scala/sttp/client3/ConvertFromFuture.scala

```diff
@@ -0,0 +1,13 @@
+package sttp.client3
+
+import scala.concurrent.Future
+
+trait ConvertFromFuture[F[_]] {
```

Comment:
this is used only in the JS impl? I'd move it to JS sources & to an internal package (that's not the public API)